psh/mount/mount.c

```diff
@@ -23,34 +23,43 @@
 
 void psh_mountinfo(void)
 {
-	printf("mounts a filesystem");
+	(void)printf("mounts a filesystem");
```

Comment:
Looks good to me, apart single doubt `(void)pritf(...)`:
I'd like @agkaminski , @nalajcie  say about (MISRA 2012 rule 17.7, required, but decidable) if it's legit now for printf's common use ?
See section 6.5 `Decidability of rules`.

Reply:
Generally it's a good rule as it forces a programmer not to ignore retvals by default. Some `(void)` casts is an acceptable price to pay imho